matlab/src/cpp/arrow/matlab/error/error.h

```diff
@@ -195,6 +192,6 @@ namespace arrow::matlab::error {
     static const char* CHUNKED_ARRAY_MAKE_FAILED = "arrow:chunkedarray:MakeFailed";
     static const char* CHUNKED_ARRAY_NUMERIC_INDEX_WITH_EMPTY_CHUNKED_ARRAY = "arrow:chunkedarray:NumericIndexWithEmptyChunkedArray";
     static const char* CHUNKED_ARRAY_INVALID_NUMERIC_CHUNK_INDEX = "arrow:chunkedarray:InvalidNumericChunkIndex";
-
-
+    static const char* ARROW_NUMERIC_INDEX_ZERO_FIELDS = "arrow:index:NumericIndexWithZeroFields";
+    static const char* ARROW_INVALID_NUMERIC_FIELD_INDEX = "arrow:index:InvalidNumericFieldIndex";
```

Comment:
`INDEX_XXX = "..."` may be better because (most) other constant names have the `XXX` part in `arrow:XXX:YYY` and doesn't have `ARROW_` prefix.

Reply:
Will do!